### PR TITLE
Docs cli

### PR DESCRIPTION
**Pull request by chStaiger**
Spell checking and some wording.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,7 +74,7 @@ download(session, "/irods/path", "/other/local/path")
 ```
 
 ## Command line interface
-To simply upload or download data you do not need to write full python program, we offer a command line interface
+To simply upload or download data you do not need to write full python program, we offer a command line interface (see also [the documentation](https://ibridges.readthedocs.io/en/latest/)).
```
Comment (qubixes) [inline README.md:77]: Maybe just link the "command line interface" directly?

File: docs/source/cli.rst [hunk 2]
```diff
@@ -2,7 +2,7 @@ Command Line Interface
 ======================
 
 iBridges also has a Command Line Interface (CLI). The CLI provides an even more simplified
-interface for uploading, downloading and synchronizing. It does not provide (nor intends to) all the features
+interface for uploading, downloading and synchronising. It does not provide (nor intends to) all the features
```
Comment (qubixes) [inline docs/source/cli.rst:5]: I'm generally using American spelling, wouldn't it be "synchronizing"?

File: docs/source/cli.rst [hunk 3]
```diff
@@ -61,8 +61,23 @@ Please try to avoid spaces in collection and data object names! If you really ne
...
+
+To create a new collection in you iRODS home simply type:
+
+.. code:: shell
+
+	ibridges mkcoll "irods:~/new_collection"	
+
+Or:
+
+.. code shell
```
Comment (qubixes) [inline docs/source/cli.rst:78]:
```suggestion
.. code:: shell
```

**Review by qubixes — CHANGES_REQUESTED** [review]
Thanks for the update. I don't know whether English or American spelling should be used. I would personally go with American spelling, but English is fine as well of course. There might be other American spelling mistakes though throughout the code base.

File: docs/source/cli.rst (hunk 2 above)
Comment (chStaiger) [reply docs/source/cli.rst:5]: Yes .. done, though the link will only work once we moved the rtd to main.

Comment (chStaiger) [reply docs/source/cli.rst:5]: I am torn between two spellcheckers ... one for markdown and for jupyter notebooks ... I think most of what I wrote now is BE. Shall I switch to AE?

**Pull request merged**